src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let emptyAgain = 0;
+  let sum = 0;
+  const addAllCurry = (a) => {
+    if (a !== undefined) {
```

Comment:
You can remove this condition, just add the default value (like zero) to `a` param.

Reply:
Rename `a` to more meaningful name.